ts/features/fims/common/saga/index.ts

```diff
@@ -1,7 +1,21 @@
 import { SagaIterator } from "redux-saga";
-import { fork } from "typed-redux-saga/macro";
+import { fork, select } from "typed-redux-saga/macro";
+import { apiUrlPrefix } from "../../../../config";
+import { sessionTokenSelector } from "../../../../store/reducers/authentication";
+import { createFimsClient } from "../../history/api/client";
+import { watchFimsHistorySaga } from "../../history/saga";
 import { watchFimsSSOSaga } from "../../singleSignOn/saga";
 
+const FIMS_DEV_ENV_TOKEN = "";
+
 export function* watchFimsSaga(): SagaIterator {
+  const fimsClient = createFimsClient(apiUrlPrefix);
+
+  const fimsToken = yield* select(sessionTokenSelector);
```

Comment:
All of this will be replaced when the backend spec will be available, nonetheless, what about `sessionToken`, since `fimsToken` is ambiguous with the one we exchange with the provider?